src/interaction/mobiledraw.js

```diff
@@ -12,16 +12,6 @@ goog.require('ol.layer.Vector');
 goog.require('ol.source.Vector');
 
 
-/**
- * @enum {string}
- */
-ngeo.interaction.MobileDrawProperty = {
```

Comment:
Why do you remove that ?

Reply:
@gberaudo told me to remove the enum and use strings instead, see HipChat discussion on January 11th.

Reply:
Ok, thanks !